src/psyclone/tests/psyGen_test.py

```diff
@@ -3047,14 +3047,30 @@ def test_loop_gen_code():
     assert "DO cell=1,mesh%get_last_halo_cell(1),2" in gen
 
 
+def test_invalid_loop_annotations():
+    ''' Check that the Loop constructor validates any supplied annotations. '''
+    from psyclone.psyGen import Loop
```

Comment:
Perhaps import Loop once at the top then remove replicated imports.

Reply:
It's only imported 3 times in this file.

Reply:
Oh, OK then :-)